### ISSUE TITLE
HAVE_UMFPACK is replaced by HAVE_SUITESPARSE_UMFPACK in newer dune-istl versions.

### ISSUE THREAD

**Issue by dr-robertk** [OPM/opm-common#448]
Is it as simple as adding HAVE_SUITESPARSE_UMFPACK;
in Finddune-istl.cmake?

**Comment by akva2** [comment]
we already carry the alias for both of them afaik, ref https://github.com/OPM/opm-common/blob/master/cmake/Modules/OpmAliases.cmake

have you tested if it works?

**Comment by dr-robertk** [comment]
It failed, that's why I opened the issue. But I did not see the alias. HAVE_UMFPACK is set to 1 in config.h but HAVE_SUITESPARSE_UMFPACK does not appear. I have build all opm modules with the officially permitted way, aka cmake ../ when in build.

**Comment by akva2** [comment]
hmm, indeed, i seem to have forgotten to add the config variable. yeah, it should be fixed by doing what you suggest.

**Comment by akva2** [comment]
right, i see what happened. i only added it to the config vars for the user module (opm-simulators), but i did not added it in the general the dune-istl list.

**Comment by dr-robertk** [comment]
Ok, then I'll create a PR.
